ci-operator/config/redhat-chaos/prow-scripts/redhat-chaos-prow-scripts-main-nightly-4.16-krkn-hub-aws.yaml

```diff
@@ -42,7 +42,6 @@ tests:
     test:
     - chain: redhat-chaos-krkn-hub-etcd-tests
 zz_generated_metadata:
-  branch: main
+  branch: main-nightly-4.16-krkn-hub-aws
```

Comment:
I think our goal here is to get you more aligned with the format in [openshift-release-master__nightly-4.18.yaml](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/release/openshift-release-master__nightly-4.18.yaml).  What you had before with `branch:main` looked good,  `variant: nightly-4.16` would match up with the release standard, if you really want to differentiate the component-readiness jobs from others maybe  `variant: cr-nightly-4.16`  and then just rename the [tests]()https://github.com/openshift/release/blob/3b707bcd3bdd002241758385cce8ec22ee345bd4/ci-operator/config/redhat-chaos/prow-scripts/redhat-chaos-prow-scripts-main-nightly-4.16-krkn-hub-aws.yaml#L26 to `aws-krkn-hub`

Would that pattern be reasonable?

Reply:
Hi @neisw thanks for the feedback! Sorry for the delay, updated the PR as you suggested. PTAL when you get time. Thanks.